### ISSUE TITLE
What should be the banaction choice in the jails?

### ISSUE THREAD

**Issue by debashisparial** [ritsu/ipset-fail2ban#7]
What should be the banaction in the jails? can it be iptables-multiport, iptables-ipset-proto6 or any others?

**Comment by melroy89** [comment]
Yes see also: https://serverfault.com/questions/1082564/fail2ban-ipset-which-conf-to-use

Try to use `iptables-ipset-*` nowadays instead of this custom scripting. Fail2ban now support ipset directly.
